packages/forms/src/model.ts

```diff
@@ -211,6 +211,92 @@ function assertAllValuesPresent(control: FormGroup|FormArray, value: any): void
   });
 }
 
+/**
+ * TypedOrUntyped allows one of two different types to be selected, depending on whether the Forms
+ * Class it's applied to is typed or not.
```

Comment:
nit: since `class` doesn't have a symbol (like it has in Java, for example) I'd opt to use lowercase here:

```suggestion
 * class it's applied to is typed or not.
```